runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -165,15 +165,17 @@ include::reuse::partial$billing/pricing.adoc[tag=featureAdvReq]
   +
      - Once you enable tracing, and redeploy the application, the Runtime Manager backend starts instrumenting traces for your application.
    | - xref:cloudhub-2::ch2-monitor-apps.adoc[Monitoring Apps Deployed to CloudHub 2.0] +
-     - xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-distributed-tracing[Enable Trace Data Collection]
+     - xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-trace-data-collection[Enable Trace Data Collection]
 |CloudHub| Runtime Manager dashboards provide performance metrics for all deployed applications. | Alerts based on alert conditions on deployed applications. Standard notifications to inform you when events occur in your applications. Add a CloudHub connector to your application's flows and set up custom alerts and notifications triggered by any event.  | xref:monitoring.adoc[Monitor Applications and Servers]
 |Hybrid| Runtime Manager dashboards provide performance metrics for all deployed applications. | Alerts based on alert conditions on deployed applications. Standard notifications to inform you when events occur in your applications. Alerts triggered by events related to the servers on which they run.  | xref:monitoring.adoc[Monitor Applications and Servers]
 |Private Cloud Edition| Anypoint Platform PCE does not support Runtime Manager dashboards. | Alerts based on alert conditions on deployed applications. Standard notifications to inform you when events occur in your applications. Alerts triggered by events related to the servers on which they run. | xref:monitoring.adoc[Monitor Applications and Servers]
 |Runtime Fabric
- | The Runtime Fabric agent monitors Kubernetes Deployments labelled with `rtf.mulesoft.com/id`.
+ | - The Runtime Fabric agent monitors Kubernetes Deployments labelled with `rtf.mulesoft.com/id`. +
+    - You can enable trace data collection for your Runtime Fabric applications running on Mule version 4.6 or later. 
```

Comment:
```suggestion
    - You can enable trace data collection for your Runtime Fabric applications running on Mule 4.6 or later. 
```

Reply:
I'll leave it as is, as this line is taken from the RTF docs, and was already approved and published. :)